src/systems/ec/implementations/vanilla/packages/ec-component-card/ec-component-card-print.scss

```diff
@@ -0,0 +1,112 @@
+/**
+ * Card
+ * @define card
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-card-print(
+  $color-meta: $ecl-color-grey-100,
+  $color-description: $ecl-color-black-100,
+  $color-info: $ecl-color-grey-100
+) {
+  .ecl-card {
+    margin-top: $ecl-spacing-print-l;
```

Comment:
The rule `margin: 0` should still be applied on every component I guess...

Reply:
That was my first intention. But display of the template isn't correct in this case. And we can't add utilities just for print.